src/main/java/org/opensearch/ad/cluster/HashRing.java

```diff
@@ -243,17 +240,21 @@ private void buildCircles(Set<String> removedNodeIds, Set<String> addedNodeIds,
         try {
             DiscoveryNode localNode = clusterService.localNode();
             if (removedNodeIds != null && removedNodeIds.size() > 0) {
-                LOG.info("Remove nodes from AD version hash ring: {}", Arrays.toString(removedNodeIds.toArray(new String[0])));
+                LOG.info("Node removed: {}", Arrays.toString(removedNodeIds.toArray(new String[0])));
```

Comment:
+ here too, can this be removed? or changed to `debug` level?

Reply:
The log just print when new node joins, generally it only prints several times. We'd better keep these log to help RCA. For example, this PR helped me when I debug the null AD version issue fixed in this PR.